about.html

```diff
@@ -0,0 +1,83 @@
+    <link href='https://fonts.googleapis.com/css?family=Montserrat' rel='stylesheet'>
+    <link rel="stylesheet" href="about.css" type="text/css" media="all" />
+    <title>About Page</title>
+</head>
+<body class="body">
+  <header id="header">
+        <div class="so-container">
+        <div class="socials">
+            <img id="facebook" src="./images/socials/facebook.svg" alt="Facebook Handle.">
+            <img id="twitter" src="./images/socials/twitter.svg" alt="Twitter Handle.">
+            <ul class="nav-list">
+                <li id="top-1">English</li>
+                <li id="top-2">My Page</li>
+                <li id="top-3">Logout</li>
+            </ul>
+            </div>
+            </div>
+  <nav class="a-nav">
+     <img id="ham-menu" src="./images/ham.png" alt="Hamburger Menu" />
+     <img id="a-logo" src="./images/nba-logos/nba-logo.svg" alt="Logo">
+     <div id="this-menu">
+     <img id="a-exit" src="./images/close-button.png" alt="Exit Menu" />
+     <ul id="nav-menu">
+        <li id="home"><a class="link" href="index.html">Home</a></li>
+        <li id="menu-1"><a class="link" href="about.html">About</a></li>
+        <li id="menu-2">Program</li>
+        <li id="menu-3">Join</li>
+        <li id="menu-4">Sponsor</li>
+        <li id="menu-5">News</li>
+        <li id="menu-6">Others</li>
+      </ul>
+      </div>
+  </nav>
+  </header>
+    <section id="about">
+    <h1 class="about-title">THIS IS THE NBA FINALS 2023</h1>
+    <p id="about-text">The NBA Finals is the annual Championship series of the National Basketball Association(NBA). The Eastern and Western confernce Champions play a best-of-seven game series to determine the league Champion.The team that wins the series is awarded the Larry O'Brien Championship Trophy.
+    <p class="contact">Please Contact Us per Email for any questions you may have about the NBA<br id="web-link"><a class="ref" href="https//:www.nba.com">NBA.com</a></p>
+    </section>
+    <section id="logo-section">
+        <h3 class="logo-head">NBA LOGO</h3>
+        <p class="logo-text">Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolores odit asperiores animi totam obcaecati sunt, neque reiciendis facilis! Temporibus, modi.</p>
+        <img id="logo-img" src="./images/nba-logos/nba-logo.svg" alt="Nba logo">
+    </section>
+    <section id="prev-finals">
+        <h4 class="prev-text">See the past NBA Finals</h4>
+        <p id="prev-sum">Lorem ipsum dolor sit amet consectetur adipisicing elit. Id</p>
+        <div id="finals">
+          <div class="div-1">
+            <img id="first" src="./images/arena/nba-arena.jpeg" alt="2021 NBA Finals">
+            <p class="text-1">2021 NBA Finals</p>
+          </div>
+            <div class="div-2">
+            <img id="second" src="./images/arena/td-garden.png" alt="2020 NBA Finals">
+            <p class="text-2">2020 NBA Finals</p>
```

Comment:
- [ ] Please try to use the `background color white` instead of the background image and other brownish colors. This is because a good developer is supposed to pay attention to all details and also implement them.